app/component/ItineraryTab.js

```diff
@@ -354,6 +390,50 @@ class ItineraryTab extends React.Component {
                 />
                 {config.showRouteInformation && <RouteInformation />}
               </div>
+              {config.showCO2InItinerarySummary && co2value >= 0 ? (
+                <div className="itinerary-co2-comparison">
+                  <div className="itinerary-co2-line">
+                    <div className={cx('divider-top')} />
+                    <div className="co2-container">
+                      <div className="co2-description-container">
+                        <Icon img="icon-icon_co2_leaf" className="co2-leaf" />
+                        <span className={cx("itinerary-co2-description", { simple: useCo2SimpleDesc })}>
+                          <span aria-hidden="true">
+                            <FormattedMessage id={co2DescriptionId}
+                              defaultMessage={`CO₂ emissions for this route: ${co2value} g`}
+                              values={{
+                                co2value,
+                                carCo2Value,
+                              }}
+                            />
+                          </span>
+                          <span className="sr-only">
+                            <FormattedMessage id={`${co2DescriptionId}-sr`}
+                              defaultMessage={`Carbondioxide emissions for this route: ${co2value} g`}
+                              values={{
+                                co2value,
+                                carCo2Value,
+                              }}
+                            />
+                          </span>
+                          {config.URL.EMISSIONSINFO && (
+                            <div className='co2link'>
+                              <a href={`${config.URL.EMISSIONSINFO[currentLanguage]}`}>
+                                <FormattedMessage
+                                  id="itinerary-co2.link"
+                                  defaultMessage="Näin vähennämme päästöjä ›"
+                                />
+                              </a>
+                            </div>
+                          )}
+                        </span>
+                      </div>
+                    </div>
+                    <div className={cx('divider-bottom')} />
+                  </div>
+                </div>
+              ) :
+                (null)}
```

Comment:
`ItineraryTab` is already huge. I'd suggest to create a proper component for co2information.

Reply:
Yes that is a good point, I think we should do it